### ISSUE TITLE
DOCUMENT] GIBCT SCO - Page headings SHOULD be tagged in Section 305 PDF

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8749]
## Description

VA 508 office has identified the [Colmery Act Section 305](https://www.benefits.va.gov/gibill/docs/scotraininginstructions.pdf) PDF as having page headings that are not announce properly to screen readers. Screenshot attached below is illustrative of one page. This appears to be a common element from the source document, so tagging it should hopefully fix all of the issues.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Remediation Resources
* Run the [Common Look PDF validator](https://commonlook.com/accessibility-software/pdf-validator/) or the [Acrobat Accessibility Checker](https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html) to check for headers being tagged properly

## Acceptance Criteria

* Page headers (h2) are announced properly

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs
![Screen Shot 2020-05-05 at 9 30 31 AM](https://user-images.githubusercontent.com/934879/81080753-b8922900-8eb6-11ea-8737-7ae4291076ac.png)

**Comment by desireeturner** [comment]
@TamGIBill I believe this is content that you can update. Please let me know if it is not. Thank you!

**Comment by af-manish** [comment]
@noahgelman  <cc @ERRonat0609 @mrblanco1 > Any updates?

**Comment by noahgelman** [comment]
@af-manish Did you read my last comment? Ask @TamGIBill

**Comment by ERRonat0609** [comment]
Per email from Brian Grubb, Tammy will work on this after the first of the year (2022)

**Comment by TamGIBill** [comment]
Added alt text to images and tagged header as H2

**Comment by ERRonat0609** [comment]
@noahgelman - Tammy has made changes, can you or someone in your accessibility group please validate when you get some time? Thanks!

**Comment by mrblanco1** [comment]
Hi @noahgelman - GM! Just checking if you had time to check this ticket since Tammy made changes and it should be ready to pass. Can you please confirm? Thanks!

**Comment by af-manish** [comment]
@TamGIBill I wanted to follow-up to see if you can revisit this.

**Comment by mrblanco1** [comment]
Hi @TamGIBill - I believe this is another ticket we can close. When you get a chance, would you mind letting me know if we can go ahead and close it? Thanks!

**Comment by mrblanco1** [comment]
Tammy will follow up at the end of March 2022.

**Comment by mrblanco1** [comment]
This ticket does not belong to AFS Comparison Tool. @TamGIBill might be able to provide a solution. Thanks!

**Comment by TamGIBill** [comment]
There is a VA 508 initiative that will resolve this.


Tammy S. Hurley
Education Service
Veterans Benefits Administration
202-461-9838

From: Abe Blanco ***@***.***>
Sent: Wednesday, September 14, 2022 4:06 PM
To: department-of-veterans-affairs/va.gov-team ***@***.***>
Cc: Hurley, Tammy S. (she/her/hers)(she/they) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [department-of-veterans-affairs/va.gov-team] DOCUMENT] GIBCT SCO - Page headings SHOULD be tagged in Section 305 PDF (#8749)


This ticket does not belong to AFS Comparison Tool. @TamGIBill<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTamGIBill&data=05%7C01%7C%7Cdb995d9b5892431c174108da968c8b41%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637987827617379192%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=TsNlekdo%2Baa6gPv9LPcuJEGv9%2B7ekPfq3hOo0cd%2FMvM%3D&reserved=0> might be able to provide a solution. Thanks!

-

**Comment by artsymartha68** [comment]
Closing this issue as OCTO does not own this PDF. 

I verified this PDF is in VBA's list of documents to be remediated as part of the Web Accessibility IPT work directed by the CIO.

Fyi @SarahKay8 no work needed here, thank you.
